hw2/index.js

```diff
@@ -0,0 +1,44 @@
+function NamedOne (firstName, lastName) {
+
+    // this.lastName = lastName;
+    // this.fullName = `${this.firstName} ${this.lastName}`;
+
+    let res = {
```

Comment:
@matveyprupas Can we use `const` here?

Reply:
Yes, we can. Fixed it. 
I guess would be better to use const everywhere where we can do it, right?
Could you explain to me why const is better than let? It's because nobody won't change variables that shouldn't change?